config/xsl/evaluation/sections/rules.xsl

```diff
@@ -0,0 +1,294 @@
+                                                <xsl:otherwise>
+                                                    <xsl:value-of select="$checkSelector" />
+                                                </xsl:otherwise>
+                                            </xsl:choose>
+                                        </xsl:variable>
+
+                                        <tr class="{$stripeClass}">
+                                            <!-- Rule ID -->
+                                            <xsl:if test="position() = 1" >
+                                                <xsl:call-template name="multiRowCell">
+                                                    <xsl:with-param name="numRows" select="$numChecks" />
+                                                    <xsl:with-param name="cellContent">
+                                                        <xsl:call-template name="name-or-id">
+                                                            <xsl:with-param name="node" select="$ruleDef" />
+                                                        </xsl:call-template>
+                                                    </xsl:with-param>
+                                                    <xsl:with-param name="class" select="'key'" />
+                                                </xsl:call-template>
+                                            </xsl:if>
+                                            <!-- Check Number -->
+                                            <td class="value number">
+                                                <xsl:value-of select="number(@index) + 1" />
+                                            </td>
+                                            <!-- Enabled -->
+                                            <td class="value">
+                                                <xsl:value-of select="$checkDef/@enabled = 'true'" />
+                                            </td>
+                                            <!-- Display Value -->
+                                            <td class="value">
+                                                <xsl:value-of select="$checkDef/@displayValue = 'true'" />
+                                            </td>
+                                            <!-- Check Selector -->
+                                            <td class="value text">
+                                                <xsl:value-of select="$checkSelectorResolved" />
+                                            </td>
+                                            <!-- Check Condition -->
+                                            <td class="value text">
+                                                <xsl:value-of select="$checkDef/condition" />
+                                            </td>
+                                            <!-- Check Value -->
+                                            <td class="value text">
```

Comment:
Remove `text` to get  the value right-aligned?

Reply:
Done.